### ISSUE TITLE
Notice: Undefined property: stdClass::$detail in vendor/p7s1-ctf/7pass-php-sdk/lib/P7/SSO/ApiClient.php on line 178

### ISSUE THREAD

**Issue by meckhardt** [p7s1-ctf/7pass-php-sdk#19]
I get a warning, that `$detail` is not available.
 
```
$config = new Configuration([
...
]);

$sso = new SSO($config);
$client = $sso->accountClient($accessToken);

try {
  $client->get('/me'); 
} catch (\Exception $e) {
  // 404
  ... 
}


```

![bildschirmfoto 2016-04-26 um 10 23 22](https://cloud.githubusercontent.com/assets/827666/14811573/f4158dd2-0b98-11e6-875d-f9c23370d55d.png)

**Comment by meckhardt** [comment]
![bildschirmfoto 2016-04-26 um 10 23 22](https://cloud.githubusercontent.com/assets/827666/14811573/f4158dd2-0b98-11e6-875d-f9c23370d55d.png)

**Comment by matuszeman** [comment]
Resolved in master. Will be released in v1.2.2
